### PR TITLE
Command reference guide

### PR DESCRIPTION
**Pull request by squeemishly**

### CODE REVIEW

File: bin/restart [hunk 1]
```diff
@@ -11,4 +11,4 @@ docker network ls | grep `basename $(pwd)` | awk '{ print $1 }' | xargs docker n
 # docker images delete all 3
 docker images | grep `basename $(pwd)` | awk '{ print $1 }' | xargs docker image rm -f
 
-docker compose up -d
\ No newline at end of file
+docker-compose up -d
```
Comment (psbanka) [inline bin/restart:14]: Hmm... I'm not sure if this is the right answer. I think `docker compose` and `docker-compose` are *both* legitimate commands? should we do something like 

```bash
COMPOSE=docker-compose
if [ -x $COMPOSE ]; then
    echo "$COMPOSE exists."
else
    COMPOSE="docker compose"
    if [ -x $COMPOSE ]; then
        echo "$COMPOSE exists."
    else
        echo "Cannot find docker compose. Please make sure you have your shit set up right."
        exit 1
    fi
fi
```

File: chapters/001-getting-started/README.md [hunk 2]
```diff
@@ -236,17 +236,17 @@ After seeing the ARP packets go back and forth (which establish the ability for
 
 At this point, there are a whole bunch of manual steps to get all this going.  Now that we have proven to ourselves that we know how to do this all manually, let's automate it! We have more containers to bring up and networks to build, and doing that all by hand will be a lot of toil.
 
-We are going to use the `docker compose` command which uses the [docker-compose.yml](docker-compose.yml) file in this directory to build, configure, and start our two containers on our network.
+We are going to use the `docker-compose` command which uses the [docker-compose.yml](docker-compose.yml) file in this directory to build, configure, and start our two containers on our network.
```
Comment (psbanka) [inline chapters/001-getting-started/README.md:239]: from [this stackoverflow answer](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose)

> The docker compose (with a space) is a newer project to migrate compose to Go with the rest of the docker project. This is the v2 branch of the [docker/compose](https://github.com/docker/compose/tree/v2) repo. It's been first introduced to Docker Desktop users, so docker users on Linux didn't see the command. In addition to migrating to Go, it uses the compose-spec, and part of the rewrite may result in behavior differences.

> The original python project, called docker-compose, aka v1 of docker/compose repo, has now been deprecated and development has moved over to v2. To install the v2 docker compose as a CLI plugin on Linux, supported distribution can now install the docker-compose-plugin package. E.g. on debian, I run apt-get install docker-compose-plugin.

So I don't think this change is a good idea? We could have a caveat that if you're using `docker-compose` you should upgrade your shit?

File: chapters/command-reference-guide.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+[@bork's comic](https://wizardzines.com/comics/ping/) explaining `ping` and `traceroute`.
+
+`traceroute` is a network analysis tool that shows path a packet takes from the initiator of a request (i.e. a client) to the destination IP. This is done through sending single packets that have a preset `ttl` in the IP header. The `ttl` in IP context does not refer to a specific _time_ to live, instead, it is the maximum number of hops a packet is allowed to traverse before a router should drop it.
+
+> Example command:
+
+```bash
+traceroute 10.1.1.2
+```
```
Comment (psbanka) [inline chapters/command-reference-guide.md:95]: Should we document commands we've never used? Just askin'

**Review by psbanka — APPROVED** [review]
Good stuff; thank you! I actually think we should *not* be changing `docker compose` to `docker-compose` though

File: chapters/command-reference-guide.md (hunk 3 above)
Comment (squeemishly) [reply chapters/command-reference-guide.md:95]: What do you mean? Do you have suggestions? Or are you saying we shouldn't document `traceroute`? 

https://github.com/psbanka/build-your-own-internet/tree/main/chapters/003-traceroute-aside

File: chapters/001-getting-started/README.md (hunk 2 above)
Comment (squeemishly) [reply chapters/001-getting-started/README.md:239]: warble garble. ok. lemme roll back the last commit and let's talk about how we want to approach this on Wednesday.

File: chapters/command-reference-guide.md (hunk 3 above)
Comment (psbanka) [reply chapters/command-reference-guide.md]: OH, sorry. I forgot that we actually *did* use traceroute!

Comment (squeemishly) [reply chapters/command-reference-guide.md]: bahahahaha! I was wondering if that was the case! but I wasn't sure if there was a command that you were thinking of that would be helpful to add? 😆 I'll merge now.

**Pull request merged**